README.rst

```diff
@@ -340,9 +336,10 @@ like 'radosgw-agent'. This inclusion would be defined at the project level, like
     repos = {
         'ceph': {
             'extras': ['ceph-deploy', 'radosgw-agent'],
-            'deb': {
```

Comment:
If we remove the ``deb`` key what will the endpoint look like? We were going to use that key as the way to map the endpoint to this configuration.